plugins/VstEffect/VstEffectControls.cpp

```diff
@@ -62,7 +62,7 @@ void VstEffectControls::loadSettings( const QDomElement & _this )
 {
 	//m_effect->closePlugin();
 	//m_effect->openPlugin( _this.attribute( "plugin" ) );
-	m_effect->m_pluginMutex.lock();
+	if (!m_effect->m_pluginMutex.tryLock()) {return;}
```

Comment:
Won't this cause the plugin settings not to be loaded if the mutex can't be locked? I think this code runs on the main thread anyway so it shouldn't affect audio processing if it has to wait for the mutex.

Reply:
You're right here, probably a mistake.

Reply:
Fixed in e79aeb711d1756690ff7ceba4d1ff29e0578c227.